src/components/inputBook.js

```diff
@@ -1,65 +1,32 @@
-import React, { useState } from 'react';
-import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+// import PropTypes from 'prop-types';
```

Comment:
[ OPTIONAL ] Please remove commented code from here as it can ultimately make the whole project unclean.